frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -236,11 +237,29 @@ export function PolicyTemplateDetails(
               )
               break
             default:
```

Comment:
We likely need another case for `unknowncompliancy`  here.

Reply:
It was my original impression that the default implied UnknownCompliancy, but I think it does make sense to be specific.